go.mod

```diff
@@ -15,11 +15,11 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.5.0
 	github.com/submariner-io/admiral v0.14.0-m1
-	github.com/submariner-io/cloud-prepare v0.14.0-m0
-	github.com/submariner-io/lighthouse v0.14.0-m0
+	github.com/submariner-io/cloud-prepare v0.14.0-m1
+	github.com/submariner-io/lighthouse v0.14.0-m1
 	github.com/submariner-io/shipyard v0.14.0-m1
 	github.com/submariner-io/submariner v0.13.0-m2.0.20220608112640-7f4a6a79da0d
```

Comment:
submariner is not being updated (it should be as per the PR heading)

Reply:
I did it manually but the bot needs to be updated.